From 8c4b26840b5c9ba29f1d2e77fa45b212391b59ca Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 22 Jul 2014 14:37:17 -0400
Subject: [virt] kvm/x86: Drop useless cast

Message-id: <1406040016-3289-43-git-send-email-pbonzini@redhat.com>
Patchwork-id: 86121
O-Subject: [RHEL7 PATCH v2 042/221] KVM: x86: Drop useless cast
Bugzilla: 1116936
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mathias Krause <minipli@googlemail.com>

Void pointers don't need no casting, drop it.

Signed-off-by: Mathias Krause <minipli@googlemail.com>
Reviewed-by: Paolo Bonzini <pbonzini@redhat.com>
Signed-off-by: Gleb Natapov <gleb@redhat.com>
(cherry picked from commit 6b61edf76551c4ee3ad2e6e377bc4c23c42cedf5)
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kvm/x86.c b/arch/x86/kvm/x86.c
index 2cf2c75..9faf6d4 100644
--- a/arch/x86/kvm/x86.c
+++ b/arch/x86/kvm/x86.c
@@ -5419,7 +5419,7 @@ static struct notifier_block pvclock_gtod_notifier = {
 int kvm_arch_init(void *opaque)
 {
  int r;
- struct kvm_x86_ops *ops = (struct kvm_x86_ops *)opaque;
+ struct kvm_x86_ops *ops = opaque;
 
  if (kvm_x86_ops) {
   printk(KERN_ERR "kvm: already loaded the other module\n");
-- 
1.7.1